src/java-interop/java-interop-gc-bridge-mono.cc

```diff
@@ -772,72 +755,8 @@ get_gc_bridge_info_for_object (JavaInteropGCBridge *bridge, MonoObject *object)
 	return get_gc_bridge_info_for_class (bridge, mono_object_get_class (object));
 }
 
-typedef mono_bool (*MonodroidGCTakeRefFunc) (JavaInteropGCBridge *bridge, JNIEnv *env, MonoObject *obj, const char *thread_name, int64_t thread_id);
-
-static  MonodroidGCTakeRefFunc  take_global_ref;
-static  MonodroidGCTakeRefFunc  take_weak_global_ref;
-
-static mono_bool
-take_global_ref_java (JavaInteropGCBridge *bridge, JNIEnv *env, MonoObject *obj, const char *thread_name, int64_t thread_id)
-{
-	MonoJavaGCBridgeInfo    *bridge_info    = get_gc_bridge_info_for_object (bridge, obj);
-	if (bridge_info == NULL)
-		return 0;
-
-	jobject weak;
-	mono_field_get_value (obj, bridge_info->weak_handle, &weak);
-
-	jobject handle  = env->CallObjectMethod (weak, bridge->WeakReference_get);
-	log_gref (bridge, "*try_take_global_2_1 obj=%p -> wref=%p handle=%p\n", obj, weak, handle);
-
-	if (handle) {
-		jobject h   = env->NewGlobalRef (handle);
-		env->DeleteLocalRef (handle);
-		handle      = h;
-		java_interop_gc_bridge_gref_log_new (bridge, weak, get_object_ref_type (env, weak),
-				handle, get_object_ref_type (env, handle), thread_name, thread_id, "take_global_ref_java");
-	}
-	java_interop_gc_bridge_weak_gref_log_delete (bridge, weak, get_object_ref_type (env, weak), thread_name, thread_id, "take_global_ref_java");
-	env->DeleteGlobalRef (weak);
-	weak        = NULL;
-	mono_field_set_value (obj, bridge_info->weak_handle, &weak);
-
-	mono_field_set_value (obj, bridge_info->handle, &handle);
-
-	int type    = JNIGlobalRefType;
-	mono_field_set_value (obj, bridge_info->handle_type, &type);
-
-	return handle != NULL;
-}
-
-static mono_bool
-take_weak_global_ref_java (JavaInteropGCBridge *bridge, JNIEnv *env, MonoObject *obj, const char *thread_name, int64_t thread_id)
-{
-	MonoJavaGCBridgeInfo    *bridge_info    = get_gc_bridge_info_for_object (bridge, obj);
-	if (bridge_info == NULL)
-		return 0;
-
-	jobject handle;
-	mono_field_get_value (obj, bridge_info->handle, &handle);
-
-	jobject weaklocal   = env->NewObject (bridge->WeakReference_class, bridge->WeakReference_init, handle);
-	jobject weakglobal  = env->NewGlobalRef (weaklocal);
-	env->DeleteLocalRef (weaklocal);
-
-	log_gref (bridge, "*take_weak_2_1 obj=%p -> wref=%p handle=%p\n", obj, weakglobal, handle);
-	java_interop_gc_bridge_weak_gref_log_new (bridge, handle, get_object_ref_type (env, handle),
-			weakglobal, get_object_ref_type (env, weakglobal), thread_name, thread_id, "take_weak_global_ref_2_1_compat");
-
-	java_interop_gc_bridge_gref_log_delete (bridge, handle, get_object_ref_type (env, handle), thread_name, thread_id, "take_weak_global_ref_2_1_compat");
-	env->DeleteGlobalRef (handle);
-
-	mono_field_set_value (obj, bridge_info->weak_handle, &weakglobal);
-
-	return 1;
-}
-
 static mono_bool
-take_global_ref_jni (JavaInteropGCBridge *bridge, JNIEnv *env, MonoObject *obj, const char *thread_name, int64_t thread_id)
```

Comment:
I would prefer not changing these method names & related strings, as:

 1. The same names are used in dotnet/android, and consistency is "nice"
 2. The strings are present in `tools/logcat-parse` unit tests (`tests/logcat-parse-Tests`), which implies (a) they've been around awhile, and (2) I'm not immediately sure what'll happen to logcat-parse we change them.  (Hopefully nothing! But…)